hr_payroll_account_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "HR Payroll Account Operating Unit",
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Eficent Business and IT Consulting Services S.L., "
```

Comment:
```suggestion
    "author": "ForgeFlow S.L., "
```

Can you do this also in the Copyrights? You can use a separate commit. Thank you!

Reply:
@AaronHForgeFlow It's done now